requests/census_household_gb_eng.json

```diff
@@ -34,15 +34,11 @@
                 "last-name": "Bloggs"
             }
         },
-        {
-            "method": "GET",
-            "url": "/questionnaire/anyone-else-list-collector/"
-        },
         {
             "method": "POST",
-            "url": "/questionnaire/anyone-else-list-collector/",
+            "url": "/questionnaire/anyone-else-driving-question/",
             "data": {
-                "anyone-else-answer": "Yes, I want to add {ordinality} person"
+                "anyone-else-answer": "Family members and partners"
```

Comment:
Got the following error when running against your schema changes:
```bash
"POST","/questionnaire/anyone-else-driving-question/","Expected a (302) but got a (200) back when posting page: /questionnaire/anyone-else-driving-question/ with data: {'anyone-else-answer': 'Family members and partners', 'csrf_token': 'ImI3YzBmMjQxODZjZmJiM2Q1Nzc5MGJjMmE2NjhmMjk0MWQwMWU1OGQi.XsK02A.vURe9_jeLSBJxSg1NZDKhwqplRs'}",1
```
Fixed the above by changing the answer id from `anyone-else-answer` to `anyone-else-driving-question-answer` to match the schema change. However, then it failed on the accomodation section:
```bash
"GET","/questionnaire/sections/accommodation-section","Expected a (200) but got a (302) back when getting page: /questionnaire/sections/accommodation-section",1
```

Reply:
I am not getting the accommodation-section error locally...

Reply:
I am getting the error. With your schema branch I am getting an empty summary element in accommodation, which with Mebins change on summary, means you won't be able to get that summary.

Reply:
I am getting the error. I am get an empty summary element in accommodation when I build on your schema branch. When used against the latest master it will fail due to the story Mebin merged regarding showing summaries on completion

Reply:
Working now because the schemas repo is up to date with master..